README.md

```diff
@@ -43,6 +43,36 @@ yarn dev
 rails s
 ```
 
+## Postgres setup
+
+```bash
+brew services start postgresql
+```
+
+# In the database.yml change development configuration to:
+
+development:
+adapter: postgresql
+database: aircasting_development
+username: root
+password:
```

Comment:
This is included in database.yml.example. One must make sure they have the file copied and renamed to database.yml.

Reply:
Oh, I see, you meant the username change. Maybe we can just include that in example?